049-SQLDedicatedPoolPerf/Coach/README.md

```diff
@@ -0,0 +1,51 @@
+# What The Hack - Dedicated SQL Pool performance best practices - Coach Guide
+
+## Introduction
+This "What the Hack" enables attendees to understand best practices developing, maintaining, and optimizing performance for their Synapse Dedicated SQL Pool. This hack simulates a real-world scenario where a cycle manufacturer company develops its new data warehouse leveraging Synapse Dedicated Sql pool. The goal is to apply all best practices to avoid performance bottlenecks during day by day activities. 
```

Comment:
This paragraph repeats the "intro" section from the homepage (hack description) of the hack.  Please remove this text and replace with the boiler-plate text from the template as follows:

"Welcome to the coach's guide for the SQL Dedicated Pool Performance What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.

NOTE: If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)"